.github/actions/cargo-semver-checks/action.yml

```diff
@@ -0,0 +1,24 @@
+name: "Run cargo semver-checks"
+description: "Install and run the cargo semver-checks tool"
+inputs:
+  args:
+    required: true
+    description: "The arguments to pass to `cargo semver-checks`"
+runs:
+  using: "composite"
+  steps:
+    - run: wget -q -O- https://github.com/obi1kenobi/cargo-semver-checks/releases/download/v0.18.3/cargo-semver-checks-x86_64-unknown-linux-gnu.tar.gz | tar -xz -C ~/.cargo/bin
```

Comment:
I haven't (yet) used it myself, but a few projects seem to have started using [taiki-e/install-action](https://github.com/marketplace/actions/install-development-tools) to fetch the latest `cargo-semver-checks` prebuilt binary automatically, without having to always keep bumping the version number in the `wget`.

It's all the same to me, I just thought it might be worth a look in case you haven't already heard of it.

Reply:
I'd rather bump the version to be honest. Our CI has _a lot_ of moving parts so I prefer explicit control over the different versions of things interacting with each other.

My overarching goal is to optimize for development velocity: I don't mind sending a few more PRs if the overall turn-around time of a PR is super short. That also gives us a clean track record of what has changed and issues that come up can (hopefully) be fixed quicker :)

Reply:
Makes sense!